api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -105,6 +105,8 @@ public String freeTierBillingAccountName() {
     public Double defaultFreeCreditsDollarLimit;
     // Thresholds for email alerting based on free tier usage, by cost
     public ArrayList<Double> freeTierCostAlertThresholds;
+    // The contact email from Sarahsoft for billing account setup
```

Comment:
[just passing by] at least one of these is a typo.  What is the name of the company?

Reply:
ah,  that is a typo, it should be carasoft. The htlm body, and content is still waiting on review, not finalized yet.